be/src/Project/Services/DownloadCenter/DownloadCenter.Api/Services/Categories/CategoriesService.cs

```diff
@@ -145,59 +145,35 @@ public async Task<PagedResults<IEnumerable<CategoryServiceModel>>> GetAsync(GetC
                 pagedResults = categories.PagedIndex(new Pagination(categories.Count(), model.ItemsPerPage.Value), model.PageIndex.Value);
             }
 
-            var pagedCategoriesServiceModel = new PagedResults<IEnumerable<CategoryServiceModel>>(pagedResults.Total, pagedResults.PageSize);
+            var translations = _context.DownloadCenterCategoryTranslations.Where(x => pagedResults.Data.Select(y => y.Id).Contains(x.CategoryId)).ToList();
 
-            var categoriesItems = new List<CategoryServiceModel>();
+            var parentCategoriesTranslations = _context.DownloadCenterCategoryTranslations.Where(x => pagedResults.Data.Select(y => y.ParentCategoryId).Contains(x.CategoryId)).ToList();
```

Comment:
**suggestion:** we should take all translations in one query as in other examples